app/_locales/fr/messages.json

```diff
@@ -31,8 +52,23 @@
     "message": "MetaMask",
     "description": "Le nom de l'application"
   },
+  "approve": {
+    "message": "Approuver"
+  },
+  "approved": {
+    "message": "Approuvé"
+  },
   "attemptingConnect": {
-    "message": "Tentative de connexion à blockchain."
+    "message": "Tentative de connexion au réseau"
+  },
+  "attemptToCancel": {
+    "message": "Tenter d'annuler ?"
```

Comment:
Can we remove the space before the question mark?

Reply:
same as above